defaults/main.yml

```diff
@@ -22,18 +13,26 @@ react_git_key_filename: "id_ed25519"
 react_remove_git_key: true
 
 # system-wide dependencies
-react_system_wide_dependencies:
+react_remote_system_wide_dependencies:
   - yarn
```

Comment:
Do you not require yarn to be installed locally when building locally?

Reply:
when building locally, node and yarn are a pre-requisite, that should be set up before hand.
I did not have any good idea on how to include tasks to set them up in this role(i.e. for local builds).
for admin host, I have installed yarn as a global npm package after having installed node using nvm.
- problem is that ansible threw an error when I tried to add an include_role task that installs nvm and that is delegated to localhost.

Reply:
The wording looks okay but I suppose it requires an actual test on my end to know for sure that there is absolutely a need to stick to the previous variable.